src/CoreBluetooth/CBPeer.cs

```diff
@@ -10,8 +10,20 @@
 namespace CoreBluetooth {
 	public partial class CBPeer  {
 #if !NET
+#if NET
+		[SupportedOSPlatform ("ios8.0")]
```

Comment:
This one looks wrong, but here's what CBPeer has:

```
	[iOS (8, 0)]
	[Mac (10,13)]
	[Watch (4,0)]
	[BaseType (typeof (NSObject))]
	[DisableDefaultCtor] // CBPeer.h: - (instancetype)init NS_UNAVAILABLE;
	interface CBPeer : NSCopying {
```

So technically in iOS 8, Deprecated in iOS 9, Obsolete in iOS 9. What shall we do here?

Reply:
Unsupported I'd say since the iOS version that we support in .net is 11

Reply:
The whole property is inside a `!NET`, so it doesn't matter what we do, it won't have any effect. You can just skip the entire file.